test/common.py

```diff
@@ -50,6 +50,22 @@ def check_events_from_splunk(index="circleci_events",
 
     return events
 
+def check_metrics_from_splunk(index="circleci_metrics",
+                             start_time="-1h@h",
+                             end_time="now",
+                             url="",
+                             user="",
+                             password="",
+                             dimension="",
+                             metric_name=""):
+    '''
+    send a search request to splunk to check for values associated with a given metric and dimension
```

Comment:
send an search api request to splunk to check for values associated with a given metric and dimension